src/main/java/org/sonar/squidbridge/api/CheckMessage.java

```diff
@@ -103,7 +83,6 @@ public String getDefaultMessage() {
     return messageArguments;
   }
 
-  @Override
   public String getText(Locale locale) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "locale". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1172)